terraso_backend/apps/soil_id/migrations/0023_usermatchrating.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 5.2.6 on 2025-09-23 19:03
+
+import django.db.models.deletion
+import rules.contrib.models
+import uuid
+from django.db import migrations, models
+
+def update_rows(apps, schema_editor):
+    # Read the old selected_soil_id
+    # Set that soil id to SELECTED in the new field
+    SoilMetadata = apps.get_model("soil_id", "SoilMetadata")
+    UserMatchRating = apps.get_model("soil_id", "UserMatchRating")
+
+    for soil_metadata in SoilMetadata.objects.all():
+        old_selected_soil_id = soil_metadata.selected_soil_id
+        if (old_selected_soil_id):
+            UserMatchRating.objects.create(
+                soil_metadata=soil_metadata,
+                match_id=old_selected_soil_id,
+                user_rating="SELECTED"
+            )
```

Comment:
hrmph: since this is running at migration time instead of when the feature flag gets flipped on for prod users, it means the initial state of people's `UserMatchRatings` will reflect what soil they had selected when the migration ran. but we don't have control over when an individual users' feature flag gets flipped, it's just whenever their app updates 🤔